doc/user_manual/model.tex

```diff
@@ -1237,18 +1237,6 @@ \subsubsection{Method: \texttt{def run}}
 \input{postprocessor.tex}
 
 
-\paragraph{Method: dataObjectLabelFilter}
-The \xmlNode{HSPS} Post-Processor performs a filtering of the dataObject. This particular filtering is based on the labels generated by any clustering algorithm.
```

Comment:
Is this information available somewhere else or is it outdated? (As in why is it removed?)

Reply:
I don't recall; I will look into it.

Reply:
Oh, yes, this appeared in two places in the input manual, and on instruction from Diego I removed it from this one.